.devcontainer/frontend/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
Is the expectation that there'd be one devcontainer config for all development within this directory?

Reply:
I've been going back and forth on how we can best structure it - I'm thinking one devcontainer for all development in the `frontend` directory may make sense by having the entire toolset available within the turborepo based directory. (Other options were one devcontainer per project which seemed overkill and using the `code-dot-org` one that we did a while back)

The devcontainer here is experimental but the intention is to have it for `frontend` directory for now.

Reply:
I'd recommend we keep it for now since it needs to be there in order for development to work in the first place. That being said, I think we'll  use the `cms` branch so it won't land in `staging` just yet until we solicit more feedback from other engineering teams.

Reply:
> "In order for development to work in the first place"

Is the implication that all development must take place in a devcontainer?